app/bundles/LeadBundle/Views/Integration/index.html.php

```diff
@@ -38,6 +38,10 @@
                     <dd><?php echo $view['date']->toText($details['date_added'], 'UTC', 'Y-m-d H:i:s', true); ?></dd>
                     <dt><?php echo $view['translator']->trans('mautic.integration.last_sync_date'); ?></dt>
                     <dd><?php echo $view['date']->toText($details['last_sync_date'], 'UTC', 'Y-m-d H:i:s', true); ?></dd>
+
+                    <?php foreach (($details['extra-data'] ?? []) as $extraData): ?>
+                        <dd><?php echo $extraData; ?></dd>
```

Comment:
Should be better to use `$key => $value` to preserve `<dt>` `<dd>` structure of 

![image](https://user-images.githubusercontent.com/462477/129433333-ab83cb9f-c957-42ce-a24d-a8128ad7305a.png)

Reply:
True, I'll make the changes